ext/standaloneusers/CRM/Standaloneusers/Upgrader.php

```diff
@@ -44,21 +45,52 @@ public function onInstall() {
    */
   public function postInstall() {
 
+    // Ensure users can login with username/password via authx.
     Civi::settings()->set('authx_login_cred', array_unique(array_merge(
       Civi::settings()->get('authx_login_cred'),
       ['pass']
     )));
 
-    $users = \Civi\Api4\User::get(FALSE)->selectRowCount()->execute()->countMatched();
-    if ($users == 0) {
-      CRM_Core_DAO::executeQuery('DELETE FROM civicrm_uf_match');
-    }
+    $this->createPasswordResetMessageTemplate();
 
     // `standaloneusers` is installed as part of the overall install process for `Standalone`.
     // A subsequent step will configure some default users (*depending on local options*).
     // See also: `StandaloneUsers.civi-setup.php`
   }
 
+  protected function createPasswordResetMessageTemplate() {
+
+    $baseTpl = [
+      'workflow_name' => 'password_reset',
+      'msg_title' => 'Password reset',
+      'msg_subject' => 'Password reset link for {domain.name}',
+      'msg_text' => <<<TXT
```

Comment:
Looks like it was merged but I've got a question about this text. Does it need to be wrapped in something (like `ts()`) to be available for translation, or does `MessageTemplate` handle that?

Reply:
Good point. Message templates used for automated system-workflows should be stored in the DB with the `{ts}` calls. 

For comparison, you can skim some default data:

```
 cv api4 -T MessageTemplate.get +s id,workflow_name,msg_subject
```